src/Components/Patient/DailyRoundListDetails.tsx

```diff
@@ -33,8 +33,9 @@ export const DailyRoundListDetails = (props: any) => {
 
   return (
     <Page
-      title={`Consultation Update #${id}`}
+      title={`Log Update #${id}`}
       backUrl={`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/daily-rounds`}
+      crumbsReplacements={{ ["daily-rounds"]: { name: "Log Update" } }}
```

Comment:
Like already mentioned in the previous review: Change the URL path in the AppRouter instead.